prettyprinter/src/Data/Text/Prettyprint/Doc/Render/Text.hs

```diff
@@ -56,7 +56,7 @@ import Control.Applicative
 --       (foo bar)
 --       sit amet
 renderLazy :: SimpleDocStream ann -> TL.Text
-renderLazy = TLB.toLazyText . renderSimplyDecorated TLB.fromText (pure mempty) (pure mempty)
+renderLazy = TLB.toLazyText . renderSimplyDecorated (const TLB.fromText) (pure mempty) (pure mempty)
```

Comment:
`renderLazy` should be fast. Does this change have any impact on that?

Reply:
Are there benchmarks to run somewhere? I'd hope GHC would be smart enough for this not to make any real difference.

Reply:
The `fusion` and `large-output` benchmarks both use `renderLazy`.

Reply:
Okay, doesn't look like any statistically significant difference.